apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -175,17 +185,24 @@ var TopInstructions = React.createClass({
    * collapsed
    */
   getMinHeight() {
-    const buttonHeight = $(ReactDOM.findDOMNode(this.refs.collapser)).outerHeight(true);
+    const collapseButtonHeight = getOuterHeight(this.refs.collapser, true);
+    const scrollButtonsHeight = this.refs.scrollButtons ?
+        this.refs.scrollButtons.getMinHeight() : 0;
+
     const minIconHeight = this.refs.icon ?
-      $(ReactDOM.findDOMNode(this.refs.icon)).outerHeight(true) : 0;
+      getOuterHeight(this.refs.icon, true) : 0;
     const minInstructionsHeight = this.props.collapsed ?
-      $(ReactDOM.findDOMNode(this.refs.instructions)).outerHeight(true) : 0;
+      getOuterHeight(this.refs.instructions, true) : 0;
 
     const domNode = $(ReactDOM.findDOMNode(this));
     const margins = domNode.outerHeight(true) - domNode.outerHeight(false);
 
-    return Math.max(buttonHeight, minIconHeight, minInstructionsHeight) +
-      RESIZER_HEIGHT + margins;
+    const leftColHeight = minIconHeight,
+        middleColHeight = minInstructionsHeight,
+        rightColHeight = collapseButtonHeight + scrollButtonsHeight;
```

Comment:
we dont generally use this style of declaring multiple variables with a single var/const. I don't know that there's any good reason for it, but it's nice to be consistent and I might just use three consts